astropy/units/physical.py

```diff
@@ -3,13 +3,13 @@
 """Defines the physical types that correspond to different units."""
 
 import numbers
-import sys
 import warnings
 
 from . import core
 from . import si
 from . import astrophys
 from . import cgs
+from . import imperial  # Need this for backward namespace compat # noqa
```

Comment:
It's complex enough keeping a link to the story here I think is worth it
```suggestion
from . import imperial  # Need this for backward namespace compat, see issues 11975 and 11977  # noqa
```

Reply:
OK, I accepted this suggestion but I don't want to rerun the CI. Please see the CI from the first commit. Thanks!